testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/subsystem/SimpleServletWithScope.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright (c) 2023, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.wildfly.test.integration.elytron.oidc.client.subsystem;
+
+import jakarta.annotation.security.DeclareRoles;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.HttpConstraint;
+import jakarta.servlet.annotation.ServletSecurity;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import org.wildfly.security.http.oidc.OidcSecurityContext;
+import org.jboss.as.test.integration.security.common.servlets.SimpleServlet;
+import org.wildfly.test.integration.elytron.oidc.client.KeycloakConfiguration;
+
+
+/**
+ * Modified version of {@link SimpleServlet}. Checks token to find claims obtained using scope values in OIDC authentication
+ *
+ * @author Prarthona Paul
+ */
+@DeclareRoles({ SimpleServletWithScope.ALLOWED_ROLE })
+@ServletSecurity(@HttpConstraint(rolesAllowed = { SimpleServletWithScope.ALLOWED_ROLE }))
+@WebServlet(SimpleServletWithScope.SERVLET_PATH)
+public class SimpleServletWithScope extends SimpleServlet{
+
+    /** The serialVersionUID */
+    private static final long serialVersionUID = 1L;
+    public static final String SERVLET_PATH = "/SimpleServletWithScope";
+    public static final String ALLOWED_ROLE = KeycloakConfiguration.JBOSS_ADMIN_ROLE;
```

Comment:
I think we can use JBOSS_ADMIN_ROLE directly here instead of assigning it to a new constant variable.